frontend/src/pages/learningCenter/LearningCenterDataView.tsx

```diff
@@ -94,9 +98,10 @@ const LearningCenterDataView: React.FC<LearningCenterDataViewProps> = React.memo
       return (
         <div className={listViewClasses} aria-label="Simple data list example">
           <LearningCenterListHeader />
-          {filteredDocApps.map((doc) => (
+          {filteredDocApps.map((doc, idx) => (
             <OdhDocListItem
               key={`${doc.metadata.name}`}
+              itemId={`${doc.spec.appDisplayName}-${idx}`}
```

Comment:
I think it doesn't hurt to use `doc.spec.appDisplayName` here though I prefer `doc.metadata.name` or `doc.metadata.uid`. The code LGTM! Thanks @jenny-s51